src/OrleansRuntime/Streams/QueueBalancer/StreamQueueBalancerFactory.cs

```diff
@@ -21,7 +22,7 @@ internal class StreamQueueBalancerFactory
         /// <param name="siloMaturityPeriod">Maturity Period of a silo for queue rebalancing purposes</param>
         /// <returns>Constructed stream queue balancer</returns>
         public static IStreamQueueBalancer Create(
-            StreamQueueBalancerType balancerType,
+            string balancerType,
             string strProviderName,
             ISiloStatusOracle siloStatusOracle,
```

Comment:
And I assume we were talking about `IStreamQueueBalancerFactory `interface, because this diff lie on the internal `StreamQueueBalancerFactory `class, which doesn't implement `IStreamQueueBalancerFactory `and is a completely different thing. 

And if we are indeed talking about the internal `StreamQueueBalancerFactory`, I'm not opposed to create this internal factory through DI, just pointing out that this internal factory class is a static class with a static method and used just in one place, not sure how valuable we is that change.